<launch>
    <node name="battery_pose_vel_node" pkg="data_monitor" type="battery_pose_vel_node" output="screen" />
</launch>
